pkg/vsphere/tasks/waiter.go

```diff
@@ -63,22 +62,45 @@ func Wait(ctx context.Context, f func(context.Context) (Waiter, error)) error {
 //       return vm.Reconfigure(ctx, config)
 //    })
 func WaitForResult(ctx context.Context, f func(context.Context) (ResultWaiter, error)) (*types.TaskInfo, error) {
-	task, err := f(ctx)
+	r := rand.New(rand.NewSource(time.Now().UnixNano())) //creates a more unique random
```

Comment:
@dougm we may not need it. but it does provide more randomness per run. or does the default `rand` already do that?